### PR TITLE
using multiplatform for package install if platform_family supports it

### PR DESCRIPTION
**Pull request by rshade**
Signed-off-by: Richard Shade <rshade@rightscale.com>

### Description
This PR enables multi_package for node['php']['packages'] if the platform_family supports it. SPEED. 

### Issues Resolved
No Issues Resolved

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: recipes/package.rb [hunk 1]
```diff
@@ -60,6 +60,12 @@
       action :install
       options node['php']['package_options']
     end
+  end unless %w(rhel debian).include?(node['platform_family'])
```
Comment (tas50) [inline recipes/package.rb]: suse and amazon here too

File: recipes/package.rb [hunk 2]
```diff
@@ -60,6 +60,12 @@
...
+
+  package node['php']['packages'] do
+    action :install
+    options node['php']['package_options']
+    only_if { %w(rhel debian).include?(node['platform_family']) }
```
Comment (tas50) [inline recipes/package.rb]: amazon and suse here

**Comment by tas50** [comment]
@rshade Can you squash this down to a single commit too

**Comment by rshade** [comment]
@tas50 everything is updated

**Pull request merged**